corelib-schemaorg/src/main/java/eu/europeana/corelib/edm/model/schemaorg/SchemaOrgConstants.java

```diff
@@ -63,6 +65,10 @@
 
     public static final String PROPERTY_DEATH_DATE = "deathDate";
 
+    public static final String PROPERTY_BEGIN_DATE = "beginDate";
```

Comment:
please replace this class with the version from develop branch, these properties are not mapped to schema.org according to the mapping file

Reply:
According to the previous comment here I just added one more constant: `public static final String TYPE_TIMESPAN = "Timespan"`. Someone can add more in the future.

Reply:
according to github diff more constants were added in this class, please revert back this file as none of the constants are needed for schema.org serialization of timespans. Even the Timespan constant should not be used.

Reply:
I have removed the Timespan constant as well.